types/src/ledger_info.rs

```diff
@@ -373,6 +380,192 @@ impl LedgerInfoWithPartialSignatures {
     }
 }
 
+/// Contains the ledger info and partially aggregated signature from a set of validators, this data
+/// is only used during the aggregating the votes from different validators and is not persisted in DB.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct LedgerInfoWithMixedSignatures {
```

Comment:
I don't have a strong opinion here. The code is ideally supposed to work even if the OSV flags are disabled. In that case, we only store the verified signatures in this data structure. I just thought it could be a little confusing to use the name `LedgerInfoWithUnverifiedSignatures`. If you prefer that name, I could change it too.

Reply:
we can rename the `LedgerInfoWithPartialSignatures` to `LedgerInfoWithVerifiedSignatures` then this can become `LedgerInfoWithUnverifiedSignatures`

Reply:
Done.